src/preamble.js

```diff
@@ -932,6 +932,18 @@ function stackTrace() {
 }
 {{{ maybeExport('stackTrace') }}}
 
+function isAbsoluteUrl(url) {
+  return /^(?:[a-z]+:)?\/\//i.test(url);
```

Comment:
Absolute paths in Node.js don't start with `//`, so this will always return `false`:
```
>> /^(?:[a-z]+:)?\/\//i.test('/absolute/path/on/server.js')
false
```

Reply:
Good point. Added a test for this scenario as well and fixed the regex.